src/wizard/dotnet/xamarin.md

```diff
@@ -0,0 +1,106 @@
+---
+name: Xamarin
+doc_link: https://docs.sentry.io/platforms/dotnet/guides/xamarin/
+support_level: alpha
+type: library
+---
+
+## Install the NuGet package
+
+```shell
+# Using Package Manager
+Install-Package Sentry.Xamarin.Forms -Version 1.0.0-alpha.2
+
+# Or using .NET Core CLI
+dotnet add package Sentry.Xamarin.Forms -v 1.0.0-alpha.2
+```
+
+<div class="alert alert-info" role="alert"><h5 class="no_toc">Using .NET Framework prior to 4.6.1?</h5>
+    <div class="alert-body content-flush-bottom">
+        <a href="https://docs.sentry.io/clients/csharp/">Our legacy SDK</a> supports .NET Framework as early as 3.5.
+    </div>
+</div>
+
+## Initialize the SDK
+
+Initialize the SDK as early as possible, like in the constructor of the `App`, and Add `SentryXamarinFormsIntegration` as a new Integration to `SentryOptions`:
+
+
+### Android
+Initialize the SDK on your MainActivity.
+
+```csharp
+public class MainActivity : global::Xamarin.Forms.Platform.Android.FormsAppCompatActivity
+{
+    protected override void OnCreate(Bundle savedInstanceState)
+    {
+        SentrySdk.Init(o =>
+        {
+			// Add the following line:
+            o.Dsn = "___PUBLIC_DSN___";
+            o.CacheDirectoryPath = Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData);
+            o.AddIntegration(new SentryXamarinFormsIntegration());
```

Comment:
Now that I see this, a proposed new API could be done with extension methods like:

```suggestion
            o.AddXamarinFormsIntegration();
```